### ISSUE TITLE
Tests: Convert Trigger Button Tests

### ISSUE THREAD

**Issue by AAlviar** [infor-design/enterprise-wc#1988]
Describe the task
Convert Trigger Button tests from https://github.com/infor-design/enterprise-wc/tree/main/tests-to-convert/ids-trigger-button

Additional context
N/A

**Comment by tmcconechy** [comment]
Yeah i think you can do these together. What happened was we had a trigger-button seperate but i moved it into the trigger field folder.

So when you do this one you can do it the same. Just put the trigger-button tests in with ids-trigger-field in that folder and we should be good.
